subscription/views.py

```diff
@@ -1,106 +1,131 @@
 import stripe
 
 from django.conf import settings
-from django.contrib.auth.decorators import login_required
-from django.contrib.auth.models import User
-from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin
 from django.http import HttpResponse, JsonResponse
```

Comment:
Swap them with Rest Framework response. Do some research about which views can be converted to API views!